Packs/VirusTotal-Private_API/Integrations/VirusTotal-Private_API/VirusTotal-Private_API.py

```diff
@@ -221,6 +221,12 @@ def create_file_output(file_hash, threshold, vt_response, short_format):
     return entry
 
 
+def get_dbot_file_context(file_hash, dbotscore):
+
+    return [{'Indicator': file_hash, 'Type': 'hash', 'Vendor': 'VirusTotal - Private API', 'Score': dbotscore},
```

Comment:
The type should be `file` and not `hash`

Reply:
done